check-deps/Makefile.check

```diff
@@ -64,6 +64,13 @@ ifneq ($(wildcard $(srcdir)/check-deps/have_libdw),)
   COMMON_LDFLAGS += $(shell pkg-config --libs libdw 2> /dev/null || echo "-ldw")
 endif
 
+ifneq ($(wildcard $(srcdir)/check-deps/have_libunwind),)
+ifeq ($(DEBUG), 1)
+  COMMON_CFLAGS  += -DHAVE_LIBUNWIND
+  COMMON_LDFLAGS += -lunwind
```

Comment:
This should be same as {C,LD}FLAGS_have_libunwind in the check-deps/Makefile.

Reply:
Thanks. I've just updated it.